File: 1426-find-n-unique-integers-sum-up-to-zero/find-n-unique-integers-sum-up-to-zero.py
class Solution:
    def sumZero(self, n: int) -> List[int]:

        res = []
        i = 0
        while i<n//2:
            res.append(i+1)
            res.append(-(i+1))
            i+=1
        if n%2!=0:
            res.append(0)

        return res
        

File: 1426-find-n-unique-integers-sum-up-to-zero/find-n-unique-integers-sum-up-to-zero.cpp
class Solution {
public:
    vector<int> sumZero(int n) {
        vector<int>res;
        if(n%2!=0){
            int i=0;
            while(i<n/2){
                res.push_back(i+1);
                res.push_back(-(i+1));
                i++;
            }
            res.push_back(0);
            return res;
        }
        int i=0;
        while(i<n/2){
            res.push_back(i+1);
            res.push_back(-(i+1));
            i++;
        }
        return res;
    }
};